src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+
+  * **The Orchestrated**: These are the core elements of a Kubernetes system. Think of them as the individual baking stations within the bakery. Orchestrated components consist of deployments that, in turn, house pods, working together in harmony.
+
+  * **The Services and Applications**: Services and applications are the workhorses of the Kubernetes system. Think of them as the indivdual bakers running each oven. Within a Kubernetes system each pod houses one or more services and applications. The services and applocations provide the essential functionality that drives the system. This could be computation, a web application, or any other application.
+
+It's important to note that these sections interalate with each other. The cluster contains multiple orchestrated layers, and each orchestrated section consists of multiple service and application layers.
+
+##  Understand and monitor the cluster layer
+
+Imagine your Kubernetes cluster as an expansive bakery again. The bakery is equipped with multiple ovens and skilled bakers staioned on each oven. If your bakery has only two or three ovens and a few bakers, it's manageable to oversee each oven unit personally. However, when we scale up with dozens or hundreds of bakers and ovens, keeping track of every single unit becomes a near-impossible task.
+
+Similarly, in a large Kubernetes system, with numerous deployments and pods, manually monitoring each component becomes impractical. New Relic offers a more efficient approach to oversee the entire system's health and receive timely alerts when issues arise.
+
+The following steps guide your through a general monitoring strategy for your cluster:
+
+<Steps>
+    <Step>
+### Navigate to the overview dashboard
+
+Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Kubernetes > Overview Dashboard**. 
+    </Step>
+    <Step>
+
+### Triage your cluster
+
+The Kubernetes overview dashboard shows your high level data about your cluster. You can find general data such as the count of pods and services, but more importantly you can find data about the health of your cluster such as the percent of pods running, the count of failed pods, the number of container restarts, and more. Be sure to scroll down to see all the graphs available to you.
+
+<img
+    title="tutorialOverviewdashboard"
+    alt="The main overview dashboard for the kubernetes capability"
+    src={tutorialOverviewdashboard}
+/>
+
+Use this dashboard to gauge the general health of your cluster. Here are a few things to look for:
+
+* Red or yellow tiles: think of yellow tiles as warnings to keep an eye on what it monitors. For example, if you have 2 unhealthy deployments you should take note and possibly plan to troubleshoot those deployments. Think of red tiles as critical alerts. These aren't necessarily failures in your system, but you should prioritize addressing them as immediately as possible.  
```

Comment:
I think you can rework the "think of" parts and just go directly into what yellow and red tiles mean.